work/v1/types.go

```diff
@@ -94,7 +94,7 @@ type ManifestConfigOption struct {
 	// UpdateStrategy defines the strategy to update this manifest. UpdateStrategy is Update
 	// if it is not set,
 	// optional
-	UpdateStrategy *UpdateStrategy `json:"updateStrategy"`
+	UpdateStrategy *UpdateStrategy `json:"updateStrategy,omitempty"`
```

Comment:
Should this also be `// +optional`?

Reply:
Good point. It doesn't change the behavior,  but it's good to be consistent with the other fields.